lib/Tsifier.js

```diff
@@ -59,27 +59,32 @@ module.exports = function (ts) {
 		if (opts.t) { expanded.target = opts.t; }
 		opts = assign(expanded, opts);
 
-		var configFile;
-		if (fileExists(opts.project)) {
-			configFile = opts.project;
-		} else {
-			configFile = ts.findConfigFile(
-				ts.normalizeSlashes(opts.project || bopts.basedir || currentDirectory),
-				fileExists
-			);
-		}
-
 		var config;
-		if (configFile) {
-			log('Using tsconfig file at %s', configFile);
-			config = tsconfig.readFileSync(configFile);
-			config.compilerOptions = config.compilerOptions || {};
+		var configFile;
+		if (typeof opts.project === "object"){
+			log('Using inline tsconfig');
+			config = opts.project;
 			extend(config.compilerOptions, opts);
```

Comment:
There is one small change I'd like you to make before I merge this. Could you change this:

    log('Using inline tsconfig');
    config = opts.project;
    extend(config.compilerOptions, opts);

to this:

    log('Using inline tsconfig');
    config = JSON.parse(JSON.stringify(opts.project));
    config.compilerOptions = config.compilerOptions || {};
    extend(config.compilerOptions, opts);

Whether or not it's necessary to extend the project with any compiler options that happen to be specified in `opts` is debatable, I guess, but it is consistent with the other file-based usage, so that's fine. However, there's no guarantee that `compilerOptions` will be defined - some callers might just be using the `files` or `exclude` options, for example. Also, mutating passed-in data is something that I prefer to avoid, so some JSON-based cloning could be used to avoid that.

Thanks for the PR.